Sources/WebAuthn/Ceremonies/Registration/AuthenticatorSelection.swift

```diff
@@ -37,3 +40,39 @@ public struct AuthenticatorSelection: Codable, Sendable {
     }
 }
 
+extension AuthenticatorSelection {
```

Comment:
For consistency with other types, can we move the `Codable` conformance over to this line?
```suggestion
extension AuthenticatorSelection: Codable {
```